www/js/goals.js

```diff
@@ -495,16 +494,16 @@ angular.module('emission.main.goals',['emission.services', 'emission.plugin.logg
     };
 
     var users = [];
-    //var allPartyList = [];
+    var partyList = [];
+    $scope.userParty = [];
     var allUsersForLeaderBoard = function(){
         var callOpts = {'method': 'GET', 'method_url': "/api/v3/members/all",
                     'method_args': null};
         CommHelper.habiticaProxy(callOpts).then(function(response){
             console.log("Sucessfully got all the users");
             var allUsers = response.data;
             var ignoreList = ['Superb Girl','Test-em-mri','Test_em-mr','Test_berkeley',
-                        'Abcdef','admin','Ucb.sdb.android.3','Ucb.sdb.android.1',
-                        'Test', "Em Jr.#4", "Em jr#3", "Em_MR_2", ];
+                        'Abcdef','admin','Ucb.sdb.android.3','Ucb.sdb.android.1','Test', 'Test100'];
```

Comment:
These additional test accounts need to be ignored as well. Maybe I will fix that locally before pushing.